File: WebApplication/src/modules/report/models/fulfillment-of-limits-report-dto.ts
import { NomenclatureDto } from "src/infrastructure/models/nomenclature.dto";
import { FulfillmentOfLimitsInfoDto } from "./fulfillment-of-limits-info-dto";

export class FulfillmentOfLimitsReportDto {
  year: NomenclatureDto;
  yearLimit: number;
  fulfillmentOfLimits: FulfillmentOfLimitsInfoDto[] = [];
}

File: StudentCredit.Application/Reports/Dtos/FulfillmentOfLimits/FulfillmentOfLimitsReportDto.cs
using StudentCredit.Infrastructure.Helpers.Dtos;

namespace StudentCredit.Application.Reports.Dtos.FulfillmentOfLimits
{
    public class FulfillmentOfLimitsReportDto
    {
        public NomenclatureDto Year { get; set; }

        public decimal YearLimit { get; set; }

        public List<FulfillmentOfLimitsInfoDto> FulfillmentOfLimits { get; set; } = new List<FulfillmentOfLimitsInfoDto>();
    }
}
